docs/bmad/stories/48.3.multi-poi-marker.md

```
@@ -0,0 +1,771 @@
+
+**Epic:** 48 - Lagekarte für Einsatzkoordination
+**Story ID:** 48.3
+**Version:** 1.0.1
+**Created:** 2025-10-10
+**Author:** Sarah (Product Owner - BMad)
+
+---
+
+## Status
+
+**Ready for Review**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)